DEPLOYNOTES.rst

```diff
@@ -18,6 +18,13 @@ Deploy and Upgrade notes
 * Run ``python manage.py setup_site_pages`` to create stub pages for all
   site content needed for main site navigation.
 
+* The parasol package will automatically create a core if one does
+  not already exist. It does require custom settings that are contained in
+  ``solr_conf_sco`` be installed on the Solr instance under the ``configsets``
+  directory prior to deployment. It also requires that ``CONFIGSET`` name is
+  provided in the `default` dictionary under ``SOLR_CONNECTIONS``.
```

Comment:
Can you add a `cp` and/or `scp` sample command showing where to copy the files? An example would be helpful to me, and I think it would also make it clearer that a manual step is required. We should probably also put a similar line in the developer notes portion of the readme.

Reply:
I'll definitely add this. It'll need to be an `scp` to server followed by a `cp` and `chown`, so it's complicated enough I appreciate the nudge to document it.